src/pudl/transform/eia861.py

```diff
@@ -2245,13 +2216,22 @@ def utility_data(tfr_dfs):
     )
 
     # Establish columns that are nerc regions vs. rtos
-    nerc_cols = [col for col in raw_ud if "nerc_region_operation" in col]
-    rto_cols = [col for col in raw_ud if "rto_operation" in col]
-
+    nerc_cols = [col for col in raw_ud if "nerc_region_operation" in col] + [
+        "data_maturity"
+    ]
+    logger.info(f"{nerc_cols=}")
+    rto_cols = [col for col in raw_ud if "rto_operation" in col] + ["data_maturity"]
+    logger.info(f"{rto_cols=}")
+    misc_cols = [
+        col
+        for col in raw_ud
+        if "nerc_region_operation" not in col and "rto_operation" not in col
+    ]
+    logger.info(f"{misc_cols=}")
```

Comment:
Could you add a helpful message for these log statements please?

Reply:
I don't really know what's going on here either. All I did was add `data_maturity`

Reply:
Ah ok. I guess we can leave it in there then.